WORKSPACE

```diff
@@ -88,3 +88,32 @@ pip_install(
     name = "grpc_python_dependencies",
     requirements = "@com_github_grpc_grpc//:requirements.bazel.txt",
 )
+
+http_archive(
+    name = "build_bazel_rules_swift",
+    sha256 = "12057b7aa904467284eee640de5e33853e51d8e31aae50b3fb25d2823d51c6b8",
+    url = "https://github.com/bazelbuild/rules_swift/releases/download/1.0.0/rules_swift.1.0.0.tar.gz",
+)
+
+http_archive(
+    name = "rules_pods",
+    urls = ["https://github.com/pinterest/PodToBUILD/releases/download/4.1.0-412495/PodToBUILD.zip"],
+)
+
+load(
+    "@build_bazel_rules_swift//swift:repositories.bzl",
+    "swift_rules_dependencies",
+)
+load(
+    "@rules_pods//BazelExtensions:workspace.bzl",
+    "new_pod_repository",
+)
+
+swift_rules_dependencies()
+
+new_pod_repository(
```

Comment:
pls add a summary of this target (e.g. matching the cronet framework artifact as in our current cocoapod builds etc. )